ext/reflection/php_reflection.c

```diff
@@ -647,8 +647,8 @@ static zval *get_default_from_recv(zend_op_array *op_array, uint32_t offset) {
 }
 
 static int format_default_value(smart_str *str, zval *value) {
-	if (Z_TYPE_P(value) <= IS_STRING) {
-		smart_str_append_scalar(str, value, SIZE_MAX);
+	if (smart_str_append_zval(str, value, SIZE_MAX) == SUCCESS) {
+		/* Nothing to do. */
 	} else if (Z_TYPE_P(value) == IS_ARRAY) {
```

Comment:
Can this branch still be reached?

Reply:
Yes, `smart_str_append_zval()` does not handle arrays atm, amongst other things.